doc/client_lua_api.md

```diff
@@ -842,16 +842,25 @@ Please do not try to access the reference until the camera is initialized, other
     * Returns with same syntax as above
 * `get_fov()`
     * Returns:
-
+    
 ```lua
      {
          x = number,
          y = number,
          max = number,
-         actual = number
+         actual = number,
+         zoom = number
      }
 ```
 
+* `set_fov(fov_degrees)`
+	* Sets the FOV for the local player
+* `set_zoom_fov(fov_degrees)`
+	* Sets the FOV for the local player when zoomed in
+* `save_fov()`
+	* Sets the "fov" setting to `get_fov().actual`
+* `save_zoom_fov()`
+	* Sets the "zoom_fov" setting to `get_fov().zoom`
```

Comment:
Why not just make 'set' set the zoom settings permanently as with any other setting?
Why have the complexity of a temporary plus a way to save it?